src/components/UserProfile.jsx

```diff
@@ -155,57 +136,44 @@ export default function UserProfile({
-            <Card titleId="PENDING_SIGHTINGS">
-              <LinearProgress style={{ margin: '16px 32px' }} />
-            </Card>
-          ) : (
-            <SightingsCard
-              id="pending-sightings-card"
-              title={
-                someoneElse
-                  ? intl.formatMessage(
-                      { id: 'USERS_UNPROCESSED_AGS' },
-                      { name },
-                    )
-                  : intl.formatMessage({ id: 'PENDING_SIGHTINGS' })
-              }
-              columns={['individual', 'date', 'location', 'actions']}
-              sightings={agsData || []}
-              linkPath="pending-sightings"
-              noSightingsMsg={
-                someoneElse
-                  ? 'NO_PENDING_SIGHTINGS_NON_SELF'
-                  : 'NO_PENDING_SIGHTINGS'
-              }
-              loading={localAgsLoading}
-            />
-          )}
-          {localSightingsLoading ? (
-            <Card titleId="SIGHTINGS">
-              <LinearProgress style={{ margin: '16px 32px' }} />
-            </Card>
-          ) : (
-            <SightingsCard
-              id="sightings-card"
-              titleId={
-                someoneElse
-                  ? intl.formatMessage(
-                      { id: 'USERS_SIGHTINGS' },
-                      { name },
-                    )
-                  : intl.formatMessage({ id: 'SIGHTINGS' })
-              }
-              columns={['individual', 'date', 'location', 'actions']}
-              hideSubmitted
-              sightings={sightingsData || []}
-              loading={localSightingsLoading}
-              noSightingsMsg={
-                someoneElse ? 'NO_SIGHTINGS_NON_SELF' : 'NO_SIGHTINGS'
-              }
-            />
-          )}
-
+          <SightingsCard
+            id="pending-sightings-card"
+            title={
+              someoneElse
+                ? intl.formatMessage(
+                    { id: 'USERS_UNPROCESSED_AGS' },
+                    { name },
+                  )
+                : intl.formatMessage({ id: 'PENDING_SIGHTINGS' })
+            }
+            columns={['individual', 'date', 'location', 'actions']}
+            sightings={agsData || []}
+            linkPath="pending-sightings"
+            noSightingsMsg={
+              someoneElse
+                ? 'NO_PENDING_SIGHTINGS_NON_SELF'
+                : 'NO_PENDING_SIGHTINGS'
+            }
+            loading={agsLoading}
+          />
+          <SightingsCard
+            id="sightings-card"
+            title={
+              someoneElse
+                ? intl.formatMessage(
+                    { id: 'USERS_SIGHTINGS' },
+                    { name },
+                  )
+                : intl.formatMessage({ id: 'SIGHTINGS' })
+            }
+            columns={['individual', 'date', 'location', 'actions']}
+            hideSubmitted
+            sightings={sightingsData || []}
+            loading={sightingsLoading}
```

Comment:
I worry that this introduces the regression of:
> first DataDisplay was only getting rendered when there was data, so after loading was finished, so no bar was being loaded because of that

Reply:
I tested the fix! Though please feel free to double-check me

Reply:
Hm. Do you happen to remember whether you tested it before or after updating houston when Jon fixed the speed issue?
https://wildme.atlassian.net/browse/DEX-786, which was merged four days ago.

Reply:
I think I probably tested after that was fixed. But I did use the network devtools to simulate very slow connections